tests/modules/nf-core/cellranger/count/nextflow.config

```diff
@@ -25,7 +25,7 @@ process {
     }
 
     withName: CELLRANGER_COUNT {
```

Comment:
If you are updating another module it would be desired that you perform a separate PR for that, as modules are versioned with the commit IDs

Reply:
These changes are removed (by checking out master branch) and updated here instead. https://github.com/nf-core/modules/pull/2646

Thank you now I understand the review and versioning system better. It was helpful however to run unit tests for both modules via GitHub Actions. I was impressed by the automated testing system.

Reply:
sure, happy to solve any questions as well!